### PR TITLE
Friendlier post-setup page

### PR DESCRIPTION
**Pull request by johnnyaug**
Before:
image

After
image

### CODE REVIEW

**Review by itai-david — APPROVED** [review]
Thanks for this upgrade. New page looks much better :-)
Approving as not to block, but *if* you need someone to review js code, I would wait for another approval

File: webui/src/pages/setup/index.jsx
Comment (itaiad200) [inline webui/src/pages/setup/index.jsx:68]: Why not push the `variant="outline-dark"` to the css if it's shared between the 2 usages?

File: webui/src/pages/setup/index.jsx
Comment (itaiad200) [inline webui/src/pages/setup/index.jsx:78]: This changes the current behaviour by creating a new tab, right?

File: webui/src/pages/setup/index.jsx
Comment (itaiad200) [inline webui/src/pages/setup/index.jsx:107]: Undo

File: webui/src/styles/globals.css
Comment (itaiad200) [inline webui/src/styles/globals.css:323]: Are these changes related?

File: webui/src/lib/components/controls.jsx
Comment (itaiad200) [inline webui/src/lib/components/controls.jsx:221]: I don't understand where you're passing the `...rest`.

File: webui/src/pages/setup/index.jsx
Comment (itaiad200) [inline webui/src/pages/setup/index.jsx:89]: Minor: path is valid for POSIX only.

**Review by itaiad200 — APPROVED** [review]
Thanks for this improvement. Mainly questions and minor comments.

**Review by peacing — APPROVED** [review]

File: webui/src/pages/setup/index.jsx
Comment (johnnyaug) [reply webui/src/pages/setup/index.jsx:68]: The variant is a react attribute, you can't apply it using CSS

File: webui/src/pages/setup/index.jsx
Comment (johnnyaug) [reply webui/src/pages/setup/index.jsx:78]: Yes, good catch! I was thinking it's better that to have the credentials still available in the previous tab.

**Pull request merged**
